timeless/poster/api.py

```diff
@@ -23,40 +24,44 @@ def locations(self):
         :return:
             Location data
         """
-        return self.__send(method=self.__GET, action="clients.getLocations").json()
+        return self.send(method=self.GET, action="clients.getLocations").json()
 
-    def __send(self, **kwargs):
+    def send(self, **kwargs):
         """Sends http request for specific poster action
 
         :return: response
         """
         response = requests.request(
             kwargs.get("method"),
             urljoin(self.url, kwargs.get("action", "")),
-            params={"format": kwargs.get("format", "json"), "token": kwargs.get("token", self.token)})
+            params=kwargs
+        )
         response.raise_for_status()
         return response
 
 
 class Authenticated(Poster):
 
-    origin = ""
-
     def __init__(self, **kwargs):
-        self.origin = kwargs.get("origin", "")
-        self.acount = kwargs.get("client_id", 0)
+        super().__init__(**kwargs)
 
-    def token(self):
-        if self.token == "":
-            self.auth()
-        return self.token
+    def access_token(self):
+        """Fetches authorization tokens
+        
+        :return: Token retrieved from Poster api or error returned by poster pi
+        """
+        return self.send(
+            method=self.POST,
+            action="auth/access_token"
+        ).json()["access_token"]
```

Comment:
@paulodamaso As I can see we don't handle cases when json hasn't `access_token`. here will be exception. first of all we should check response status code I think